iree/base/logging.h

```diff
@@ -73,6 +74,16 @@ const int WARNING = 1;  // absl::LogSeverity::kWarning
 const int ERROR = 2;    // absl::LogSeverity::kError
 const int FATAL = 3;    // absl::LogSeverity::kFatal
 
+#ifdef __ANDROID__
+// Define equivalent android log levels to map to IREE.
+const std::map<int, int> IREE_TO_ANDROID_LEVEL{
```

Comment:
this should just be an array - having an std::map pulls in a massive amount of code and per-lookup overhead - you just want `static const int kStatusToAndroidLevel[4] = {4,5,6,6};`

You also can't define this in a header file - that would put this map in every compilation unit including the file. Instead, put it in the cc file.